backend/src/sprint/sprint.service.ts

```diff
@@ -36,6 +36,12 @@ export class SprintService {
       .getOne();
   }
 
+  async getActiveSprint():Promise<Sprint[]>{
```

Comment:
Vezano more bit na projectId in vedno vrača enega ali nobenega, ne celega seznama